app/routes/auth_routes.py

```
@@ -141,71 +96,112 @@ def register():
+
+    db.session.add(new_user)
+    db.session.commit()
+    session.clear()
+    print("User registered successfully")
+    flash("Registration successful!", "success")
+    return jsonify({"success": True, "redirect": f"/auth/{user_type}"})
+
+@bp.route('/resend-otp', methods=['POST'])
+def resend_otp():
+    data = session.get('registration_data')
+    if not data: return "Session expired", 400
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)